desktop/core/src/desktop/js/apps/editor/components/aceEditor/autocomplete/AutocompleteResults.ts

```diff
@@ -2125,7 +2126,8 @@ class AutocompleteResults {
 
       if (remainingPath.length > 0) {
         let path = remainingPath.shift();
-        if (path) {
+        // path == '' needed for phoenix empty db
+        if ((this.dialect() == PHOENIX_DIALECT && path == '') || path) {
```

Comment:
Nit: `===` instead of `==`

in js falsy values equal other falsy values when using `==`, `undefined/false/null/''/0/etc == ''` is true

Also good to turn it around, i.e. `path || (this.dialect() ...)`

Reply:
done